### PR TITLE
Show Add Imports first in suggested actions

### PR DESCRIPTION
**Pull request by chborl**
fixes https://github.com/dotnet/roslyn/issues/25470

Both Add Imports and Remove Unused Variables have the same priority, so the tie is broken by their proximity to the caret.  Thus, add imports is listed 2nd.

This PR changes AddImports priority form Medium to High.  No other extensions are using CodeActionPriority.High so AddImports will now always be listed first.

Related issue https://github.com/dotnet/roslyn/issues/20487 suggests finding a better approach than ordering by proximity since it doesn't always give the desired order.

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
@chborl Do we have some tests somewhere that grab all of our actions and confirm something like:

> No other extensions are using CodeActionPriority.High so AddImports will now always be listed first.

doesn't change in the future?

**Comment by chborl** [comment]
> @chborl Do we have some tests somewhere that grab all of our actions and confirm something like:
> 
> > No other extensions are using CodeActionPriority.High so AddImports will now always be listed first.
> 
> doesn't change in the future?

I added a follow-up issue for this.  When fixing #20487 it should be discussed if AddImports should be the only extension with High priority or if there are others.

**Comment by chborl** [comment]
@jinujoe @vatsalyaagrawal  for approval

File: src/Features/Core/Portable/AddImport/References/ProjectSymbolReference.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ protected override CodeActionPriority GetPriority(Document document)
                     if (SearchResult.DesiredNameMatchesSourceName(document))
                     {
                         // The name doesn't change.  This is a normal priority action.
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/AddImport/References/ProjectSymbolReference.cs]: this comment is incorrect.  We should document why this is 'high'.

Comment (CyrusNajmabadi) [reply src/Features/Core/Portable/AddImport/References/ProjectSymbolReference.cs]: also, the comment 5 lines above is also wrong now.

**Pull request merged**